Dockerfile

```diff
@@ -3,24 +3,21 @@ FROM alpine:latest
 # Install dependencies
 RUN apk --no-cache add ca-certificates tzdata shadow su-exec
 
-# Set the working directory
+# Set working directory
 WORKDIR /listmonk
 
-# Copy only the necessary files
-COPY listmonk .
-COPY config.toml.sample config.toml
-
-# Copy the entrypoint script
+# Copy binaries and configs
+COPY listmonk run.sh config.toml config.toml.sample queries.sql schema.sql permissions.json ./
 COPY docker-entrypoint.sh /usr/local/bin/
 
-# Make the entrypoint script executable
-RUN chmod +x /usr/local/bin/docker-entrypoint.sh
+# Copy static assets
+COPY static/ ./static/
```

Comment:
This is incorrect. These assets are bundled into the binary during the build step and have no place in the Docker image.